usbfreedom/partition.py

```
@@ -0,0 +1,284 @@
+                vendor = parts[1] if len(parts) > 1 else UNKNOWN_DEVICE_FIELD
+                model = parts[2] if len(parts) > 2 else UNKNOWN_DEVICE_FIELD
+                removable = parts[3] == '1' if len(parts) > 3 else False
+
+                return DeviceInfo(
+                    path=self.device_path,
+                    size_bytes=size_bytes,
+                    vendor=vendor,
+                    model=model,
+                    removable=removable
+                )
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)